test/unittests/lock/test_lock.py

```diff
@@ -22,33 +22,33 @@
 
 from mycroft.lock import Lock
 from mycroft.util.file_utils import get_temp_path
-from mycroft.configuration import BASE_FOLDER
```

Comment:
Does `BASE_FOLDER` need to stay for backwards compat?

Reply:
no, this does not exist in mycroft-core so it's fine